lib/parallel_cucumber_worker/cli/main.js

```diff
@@ -1,5 +1,6 @@
 var Main = function() {
   var Path = require('path');
+  var Command = require('commander').Command;
```

Comment:
@simondean Actually I removed the use of command library, and got back to the original approach while solving the issue with Before/AfterFeatures events.